src/main/java/no/nav/veilarbperson/client/pdl/domain/AdressebeskyttelseGradering.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.veilarbperson.client.pdl.domain;
+
+public enum AdressebeskyttelseGradering {
+    UGRADERT,
+    FORTROLIG,
+    STRENGT_FORTROLIG,
+    STRENGT_FORTROLIG_UTLAND;
+
+    public static AdressebeskyttelseGradering mapGradering(String gradering) {
+        switch (gradering) {
+            case "UGRADERT":
+                return UGRADERT;
+            case "FORTROLIG":
+                return FORTROLIG;
+            case "STRENGT_FORTROLIG":
+                return STRENGT_FORTROLIG;
+            case "STRENGT_FORTROLIG_UTLAND":
+                return STRENGT_FORTROLIG_UTLAND;
+            default:
+                return UGRADERT;
```

Comment:
Vi forventer ikke å få andre verdier, så her tror jeg vi bør kaste exception, for `default` case.

Reply:
Jeg tror vi bør ikke kaste Exception fordi den hinder å vise de andre opplysninger til bruker. Jeg tenker vi kan legge inn "UKJENT" kode til enum og kan håndtere slik at når ukjent verdi kommer fra PDL sender vi "UKJENT" kode til frontend som skal vise "Sparret adresse, Ukjent". I frontend den blir håndtert sånn:- 
export function graderingBeskrivelse(gradering: Gradering) {
	switch(gradering) {
		case Gradering.UKJENT:
			return 'Sperret adresse, ukjent';
		case Gradering.FORTROLIG:
			return 'Sperret adresse, fortrolig';
		case Gradering.STRENGT_FORTROLIG:
			return 'Sperret adresse, strengt fortrolig';
		case Gradering.STRENGT_FORTROLIG_UTLAND:
			return 'Sperret adresse, strengt fortrolig utland';
		case Gradering.UGRADERT:
			return null;
	}
}

Beskrivelse til disse kodene, hentet fra kodeverket unntatt UKJENT kode. De som skal ha lese tilgang de kan se familiemedlem opplysninger uansett og de som skal ikke ha tilgang, de skal kun se tekst returnert av denne funksjonen. Hvis PDL/FREG legger noe ny kode, skal di gi beskjed på forhånd, vi skal få slik endringskravene på forhånd da kan vi legge den ny kode men denne koden gir litt informasjon til veileder at bruker har sperret adresse selv om den gir ikke den spesifikke beskyttelse gradering. Da vi også får litt tid til å implementere den nye endring uten å hindre veileder fra å se opplysninger som de har tilgang til.